api/src/transactions.rs

```diff
@@ -935,8 +935,10 @@ impl TransactionsApi {
                         }
                     },
 
-                    // Deprecated. Will be removed in the future.
-                    TransactionPayload::ModuleBundle(_) => {},
+                    // Deprecated.
+                    TransactionPayload::ModuleBundle(_) => {
```

Comment:
I believe this is the entrypoint for submitting txn. (submit_transaction is its caller and the entrypoint)

Reply:
done